git_greppers/utils.sh

```
@@ -1,30 +1,300 @@
+    echo "$default_value"
+  else
+    echo "$value"
+  fi
+}
+
+# Function to get default branch name for a repository
+get_default_branch() {
+  local repo_path=$1
+  local default_branch
+  
+  cd "$repo_path" || return "main"
```

Shellcheck (0.10.0)

[error] Can only return 0-255. Other data should be written to stdout.

(SC2152)